readme.md

```diff
@@ -96,7 +96,7 @@ Currently, these are the flags you can configure:
 - `contents` - Subdirectory to publish (`.` by default).
 - `releaseDraft` - Open a GitHub release draft after releasing (`true` by default).
 - `testScript` - Name of npm run script to run tests before publishing (`test` by default).
-- `2fa` - Enable 2FA on new packages (`true` by default) (it's not recommended to set this to `false`).
+- `2fa` - Enable 2FA on new packages (`true` by default) (it's not recommended setting this to `false`).
```

Comment:
Are you sure this is correct? Sounds weird to me, but I'm not a native speaker.

Maybe this would be better:
```suggestion
- `2fa` - Enable 2FA on new packages (`true` by default) (setting this to `false` is not recommended).
```

Reply:
Your suggestion is the ideal phrasing I'd recommend, but didn't want to go all "English grammer teacher" on the doc.

The phrasing adjustment I've made is about using the gerund form (aka `-ing` words) to make the sentence more concise and natural. But your phrasing is the optimal one, as I've said :)